### PR TITLE
Add first draft of system mapping blog post

### PR DESCRIPTION
**Pull request by nonspecialist**

### CODE REVIEW

File: _posts/2016-11-27-mapping-system-coupling.markdown [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+(you know the one) went down?  Do you actually <em>know</em> what systems
+depend on it? How do the reports get from it to Marketing? In this post,
+I'll describe a simple technique I've developed to surface and share
+all this juicy, interesting information.
+
+## The Problem, Restated
+
+Systems grow and morph over time, quite naturally. This happens for a
+variety of reasons: lack of time or experience are the most common
+causes, and neither of them are cause for blame.  Even in environments
```
Comment (davidw-cevo) [inline _posts/2016-11-27-mapping-system-coupling.markdown]: Repetitive "causes" suggest "are the most common" and use "cause for blame" as is

File: _posts/2016-11-27-mapping-system-coupling.markdown [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+atmosphere of trust and openness, because if the ones who really know
+how things work fear recrimination for the _way_ those things work,
+you'll never get the real picture.
+
+There are 4 stages:
+
+### Stage 1: Capture some raw data
+
+1. Get a few knowledgeable people in a room, with a large enough
+   whiteboard (or butchers paper, or post-it notes, or whatever). This
```
Comment (davidw-cevo) [inline _posts/2016-11-27-mapping-system-coupling.markdown]: Butcher's ?

File: _posts/2016-11-27-mapping-system-coupling.markdown [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+source code for all of these examples is at [the Cevo system-map
+repository](https://github.com/cevoaustralia/system-maps) in case you'd like
+to use them as a basis for your own efforts.
+
+### A web app that also sends an email
+
+Sounds simple, right? You've got a webserver, an app server, a backend
+database. That's it? Well, sure, except for how the code gets on to the
+webserver. And where it's stored before that. And that app server, it somehow
+gets an report to the external auditor every week. How does that actually
```
Comment (davidw-cevo) [inline _posts/2016-11-27-mapping-system-coupling.markdown]: "... an report ... " *a report*

File: _posts/2016-11-27-mapping-system-coupling.markdown [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+However, some key things to note:
+
+* See that blue node with a lot of magenta lines going through it? That's
+  Daisy. Daisy is a critical single point for several finance systems
+* An awful lot of things talk through one database
+* There are synchronous couplings between systems owned and maintained by
+  different teams and departments, implying human chains of communication
+  about custodianship and negotiation for fixes and when versions change.
+
+<img src="/images/complex-example.png">
```
Comment (stevemac007) [inline _posts/2016-11-27-mapping-system-coupling.markdown]: Pushing my luck here, but is there any easy way to enable this image to open in full screen, at the page resolution it's hard to see, but the actual image does have all the details.

Comment (whatbirdisthat) [inline _posts/2016-11-27-mapping-system-coupling.markdown]: something like
```html
<a target="_blank" href="/images/complex-example.png">
    <img src="/images/complex-example.png">
</a>
```

**Pull request merged**